### ISSUE TITLE
disable menu item

### ISSUE THREAD

**Issue by joey-harward** [leinardi/FloatingActionButtonSpeedDial#86]
Is there an easy way to disable a particular menu item?  I see you can remove and replace, but it would be nice to disable menu items (still show them, but semi-transparent).  For instance, I may want to disable an item temporarily until a feature is available.  Thanks!

**Comment by leinardi** [comment]
Hi @joey-harward, unfortunately there is no such feature but probably you can achieve this manually setting the alpha after creating the item:

```
FabWithLabelView fabWithLabelView = mSpeedDialView.addActionItem(new SpeedDialActionItem.Builder(R.id.fab_custom_color, drawable).create());
fabWithLabelView.setAlpha(0.6f);
```

I cannot test this right now but if you try, please let me know if works or not.

**Comment by joey-harward** [comment]
Thank you for responding, but that does not seem to work.  I am using version 1.0.2 and addActionItem returns void so I can't set alpha :)

**Comment by leinardi** [comment]
Oh right, sorry about that. Then I guess you can still do it using `findViewById()` and the id of the action:

```
FabWithLabelView fabWithLabelView = mSpeedDialView.addActionItem(new SpeedDialActionItem.Builder(R.id.fab_custom_color, drawable).create());
findViewById(R.id.fab_custom_color).setAlpha(0.6f)
```

**Comment by joey-harward** [comment]
Yes, that does change the transparency and give the "appearance" of being disabled.  Thanks.

I was hoping, however, to be able to disable it using the menu item parameter (`android:enabled="false"`).  This would not only set it semitransparent but disable click handling for that item.

Not a bug really, more of a feature request ;)

**Comment by leinardi** [comment]
Sure I will add it to my to do list :+1:
